plugins/connection/docker.py

```diff
@@ -308,11 +330,32 @@ def put_file(self, in_path, out_path):
                 count = ' count=0'
             else:
                 count = ''
-            args = self._build_exec_cmd([self._play_context.executable, "-c", "dd of=%s bs=%s%s" % (out_path, BUFSIZE, count)])
+            if getattr(self._shell, "_IS_WINDOWS", False):
+                cmd = '''& { $strm = [Console]::OpenStandardInput(1024)
+$data = [System.Array]::CreateInstance([byte],1024)
+$fstrm = [IO.File]::OpenWrite("%s")
```

Comment:
```suggestion
$fstrm = [IO.File]::OpenWrite('%s')
```
Possibly a nit, but on the off-chance `out_path` contains `$`...

Reply:
@braintist eagle eye! 😮